sdk/tests/key-provider.test.ts

```diff
@@ -11,134 +11,134 @@ describe('KeyProvider', () => {
-            expect(transferPrivateVerifier.checksum()).equal(transferPrivateVerifierLocal.checksum());
-            expect(transferPrivateToPublicProver.checksum()).equal(transferPrivateToPublicProverLocal.checksum());
-            expect(transferPrivateToPublicVerifier.checksum()).equal(transferPrivateToPublicVerifierLocal.checksum());
-            expect(transferPublicProver.checksum()).equal(transferPublicProverLocal.checksum());
-            expect(transferPublicVerifier.checksum()).equal(transferPublicVerifierLocal.checksum());
-            expect(transferPublicToPrivateProver.checksum()).equal(transferPublicToPrivateProverLocal.checksum());
-            expect(transferPublicToPrivateVerifier.checksum()).equal(transferPublicToPrivateVerifierLocal.checksum());
-            expect(unbondPublicProver.checksum()).equal(unbondPublicProverLocal.checksum());
-            expect(unbondPublicVerifier.checksum()).equal(unbondPublicVerifierLocal.checksum());
-
-            // Ensure the recovered keys are of the correct type
-            expect(bondPublicProverLocal.isBondPublicProver()).equal(true);
-            expect(bondPublicVerifierLocal.isBondPublicVerifier()).equal(true);
-            expect(claimUnbondPublicProverLocal.isClaimUnbondPublicProver()).equal(true);
-            expect(claimUnbondVerifierLocal.isClaimUnbondPublicVerifier()).equal(true);
-            expect(feePrivateProverLocal.isFeePrivateProver()).equal(true);
-            expect(feePrivateVerifierLocal.isFeePrivateVerifier()).equal(true);
-            expect(feePublicProverLocal.isFeePublicProver()).equal(true);
-            expect(feePublicVerifierLocal.isFeePublicVerifier()).equal(true);
-            expect(joinProverLocal.isJoinProver()).equal(true);
-            expect(joinVerifierLocal.isJoinVerifier()).equal(true);
-            expect(splitProverLocal.isSplitProver()).equal(true);
-            expect(splitVerifierLocal.isSplitVerifier()).equal(true);
-            expect(transferPrivateProverLocal.isTransferPrivateProver()).equal(true);
-            expect(transferPrivateVerifierLocal.isTransferPrivateVerifier()).equal(true);
-            expect(transferPrivateToPublicProverLocal.isTransferPrivateToPublicProver()).equal(true);
-            expect(transferPrivateToPublicVerifierLocal.isTransferPrivateToPublicVerifier()).equal(true);
-            expect(transferPublicProverLocal.isTransferPublicProver()).equal(true);
-            expect(transferPublicVerifierLocal.isTransferPublicVerifier()).equal(true);
-            expect(transferPublicToPrivateProverLocal.isTransferPublicToPrivateProver()).equal(true);
-            expect(transferPublicToPrivateVerifierLocal.isTransferPublicToPrivateVerifier()).equal(true);
-            expect(unbondPublicProverLocal.isUnbondPublicProver()).equal(true);
-            expect(unbondPublicVerifierLocal.isUnbondPublicVerifier()).equal(true);
-        });
+        // it('should not fetch invalid transfer keys', async () => {
```

Comment:
I think it's cleaner to use `describe.skip` or `it.skip` to skip the tests, instead of commenting it out.